model/task/expected_duration_test.go

```diff
@@ -12,6 +13,8 @@ import (
 func TestExpectedDuration(t *testing.T) {
 	assert := assert.New(t)
 	assert.NoError(db.ClearCollections(Collection))
+	indices := []map[string]interface{}{{"name": "branch_1_build_variant_1_display_name_1_status_1_finish_time_1_start_time_1", "key": map[string]int{"branch": 1}}}
+	_ = evergreen.GetEnvironment().DB().RunCommand(context.Background(), map[string]interface{}{"createIndexes": Collection, "indexes": indices})
```

Comment:
there's actually a create index helper in the client. 

also this shouldn't error?

Reply:
I tried it but found it unnecessarily hard to use/not typesafe. I instead copied stitch's wrapper around this and called that

Reply:
I want to avoid having lots of driver wrapper code/interfaces (with some exception in anser, and even then...) I think the type safety is probably not a huge issue, but if it's really hard to use we should also open a driver ticket. 

I'm happy to merge this in the short term, but we should definitley not carry this forward long term (and so should open a ticket before merging.)

Reply:
I removed the wrapper